integration/wrappers_test.go

```diff
@@ -164,8 +193,12 @@ var _ = Describe("Wrappers Integration", func() {
 			rootCmd := &cobra.Command{}
 			backup.DoInit(rootCmd) // initialize the ObjectCount
 
-			opts, _ := options.NewOptions(rootCmd.Flags())
-			err := opts.ExpandIncludesForPartitions(connectionPool, rootCmd.Flags())
```

Comment:
At the upstream [commit](https://github.com/arenadata/gpbackup/commit/783321238283de51dd3bb9c771e450643047e052) I see, that this function was changed (two new arguments was added). Can we do the same?

Reply:
> Can we do the same?

[Yes](https://github.com/arenadata/gpbackup/pull/51/files#diff-afc5c2c573c4cffee815db38bae03c14ed1bf8ca90bc68b99ccaa6d5f4b0e418R201)